services/assistant/v1-workspace-manager.js

```diff
@@ -16,14 +16,16 @@
 
 module.exports = function (RED) {
   const SERVICE_IDENTIFIER = 'assistant',
-    OLD_SERVICE_IDENTIFIER = 'conversation';
+    OLD_SERVICE_IDENTIFIER = 'conversation',
+    AssistantV1 = require('ibm-watson/assistant/v1'),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4846228)